.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
```

Comment:
I sent you a DM about this.

- [ ] Determine if we want to add more to this yml file